backend/service-graphql/src/main/kotlin/dev/johnoreilly/confetti/backend/graphql/DataStoreDataSource.kt

```diff
@@ -44,9 +54,6 @@ class DataStoreDataSource(private val conf: String, private val uid: String? = n
         }
     }
 
-    private val _speakers by lazy {
-        datastore.readSpeakers(conf).map { it.toSpeaker() }
-    }
```

Comment:
Does this remove the caching of the speaker? Last time I checked, having per-query caching of speakers did make a difference